ipaplatform/suse/tasks.py

```diff
@@ -17,10 +18,10 @@
 
 class SuseTaskNamespace(RedHatTaskNamespace):
     def restore_context(self, filepath, force=False):
-        pass  # FIXME: Implement after libexec move
+        return True  # FIXME: Implement after libexec move
 
     def check_selinux_status(self, restorecon=paths.RESTORECON):
-        pass  # FIXME: Implement after libexec move
+        return True  # FIXME: Implement after libexec move
```

Comment:
I don't think you need to change either of these. The value isn't checked on return and the other implementations don't return a boolean either.

Reply:
This has been removed/reverted.  It was changed in anticipation it was going to cause a lint error.